templates/patterns/featured_media/pattern-featured-media.html.twig

```diff
@@ -24,7 +24,14 @@
       {% set video_alignment_class = 'order-md-1' %}
     {% endif %}
 
-    <h2 class="mb-4">{{ title }}</h2>
+    {%- if title is not empty -%}
+      {% include '@oe-bcl/heading' with {
+        title: title,
+        title_tag: title_tag|default('h2'),
+        title_link: title_link,
+        attributes: create_attribute({'class': 'mb-4'}).merge(title.attributes|default(create_attribute())),
```

Comment:
title.attributes is still nok, because title is marked as a string in the yml, and we should keep it that way.. we want to move away from these objects because they cause complications in the drupal layer that limit our solutions.

Reply:
it should be cleaned now. Only creating the attribute and adding the class.